pallets/gear/src/schedule.rs

```diff
@@ -521,18 +548,24 @@ macro_rules! cost_byte_batched {
     };
 }
 
+macro_rules! to_weight {
```

Comment:
what the reason to use weight instead of u64? and to release this as macros, if there is no places where we se proof size? we can use simple function here I think.

Reply:
> what the reason to use weight instead of u64? and to release this as macros, if there is no places where we se proof size? we can use simple function here I think.

we probably want to use proof_size in the future